src/Core/Models/Export/Cipher.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Bit.Core.Enums;
+using Bit.Core.Models.View;
+using Newtonsoft.Json;
+
+namespace Bit.Core.Models.Export
+{
+    public class Cipher
+    {
+        public Cipher() { }
+
+        public Cipher(CipherView obj)
+        {
+            OrganizationId = obj.OrganizationId;
+            FolderId = obj.FolderId;
+            Type = obj.Type;
+            Name = obj.Name;
+            Notes = obj.Notes;
+            Favorite = obj.Favorite;
+
+            if(obj.Fields != null)
+            {
+                Fields = obj.Fields.Select(f => new Field(f)).ToList();
+            }
+
+            switch(obj.Type)
+            {
+                case CipherType.Login:
+                    Login = new Login(obj.Login);
+                    break;
+                case CipherType.SecureNote:
+                    SecureNote = new SecureNote(obj.SecureNote);
+                    break;
+                case CipherType.Card:
+                    Card = new Card(obj.Card);
+                    break;
+                case CipherType.Identity:
+                    Identity = new Identity(obj.Identity);
+                    break;
+            }
+        }
+
+        public string OrganizationId { get; set; }
+        public string FolderId { get; set; }
+        public CipherType Type { get; set; }
+        public string Name { get; set; }
+        public string Notes { get; set; }
+        public bool Favorite { get; set; }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Are these ignored in the other apps when null too?

Reply:
I used the web vault export as the gold standard to mimic (so not sure about other apps)